openmetadata-ui/src/main/resources/ui/src/utils/TableProfilerUtils.ts

```diff
@@ -31,7 +34,10 @@ export const calculateRowCountMetrics = (
   const rowCountMetricData: MetricChartType['data'] = [];
 
   updateProfilerData.forEach((data) => {
-    const timestamp = customFormatDateTime(data.timestamp, 'MMM dd, hh:mm');
+    const timestamp = customFormatDateTime(
```

Comment:
@ShaileshParmar11 , can you add the unit test for customFormatDateTime with new format